de/fitting.py

```diff
@@ -1,46 +1,58 @@
 """ Methods implementing the model as a fitting process. """
 
 import numpy as np
+import pandas as pd
 from jax import grad, jit
 import jax.numpy as jnp
 from jax.scipy.special import expit
 from jax.config import config
 from scipy.optimize import minimize
-from .factorization import alpha, factorizeEstimate
+from .factorization import alpha, factorizeEstimate, cellLineComparison
+from .importData import importLINCS
 
 config.update("jax_enable_x64", True)
 
 
-def reshapeParams(p, nGenes):
+def reshapeParams(p, nGenes, nCellLines=1):
```

Comment:
I wonder if you can figure out nCellLines from the length of p? In general it's best to encode information in exactly one place.